com.github.Qv2ray.json

```diff
@@ -0,0 +1,85 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "rename-appdata-file": "qv2ray.metainfo.xml",
+    "cleanup": [
+        "/include",
+        "/lib/debug",
+        "*.la",
+        "*.a"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
+        "--socket=session-bus",
```

Comment:
Please revert this back to only specific dbus access. The whole session-bus access just to display a tray icon is a no go. We can help getting the right dbus access there if needed.

Reply:
It also affects [telegram](https://github.com/flathub/org.telegram.desktop/issues/53).

Reply:
And it only affects gnome. If I run with ```--own-name=org.kde.org.kde.StatusNotifierItem-2-1```, it will show up correctly.

Reply:
Well, you could add `--own-name=org.kde.org.kde.StatusNotifierItem-2-1` to finish-args.

Reply:
Thanks, [works as expected](https://github.com/ymshenyu/flathub/commit/d628c04deb6bf71afd178461edabe53dbea9fd5d).